docs/service-brokers/gcp-broker-service-classes/cloud-pubsub/docs/plans-details.md

```diff
@@ -0,0 +1,59 @@
+| **serviceAccount** | `string` | The GCP service account to which access is granted. | YES | - |
+| **subscription** | `object` | A subscription resource. For more information, go to the **Subscription properties** section. | NO | - |
+
+### Subscription properties
+
+These are the **Subscription** properties:
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| **ackDeadlineSeconds** | `integer` | This value is the maximum time after a subscriber receives or acknowledges the message. After that time, or before the acknowledgement, the message is outstanding and is not delivered. For pull subscriptions, this value is used as the initial value for the **ackDeadline**. To override this value for a given message, call **ModifyAckDeadline** with the corresponding **ack_id**, if you use a non-streaming pull, or send the **ack_id** in a **StreamingModifyAckDeadlineRequest** if you use a streaming pull. The minimum custom deadline you can specify is `10` seconds and the maximum is `600` seconds. For push delivery, this value is also used to set the request timeout for the call to the push endpoint. If the subscriber never acknowledges the message, the Pub/Sub system eventually redelivers the message. | NO | `10` |
+| **pushConfig** | `string` | A URL locating the endpoint to which messages are pushed. If push delivery is used with this subscription, this field is used to configure it. An empty **pushConfig** signifies that the subscriber pulls and acknowledges messages using API methods. | NO | - |
+| **pushConfig.attributes** | `object` | Endpoint configuration attributes. Every endpoint has a set of API supported attributes that you can use to control different aspects of the message delivery. The currently supported attribute is `x-goog-version`, which you can use to change the format of the pushed message. This attribute indicates the version of the data expected by the endpoint. This controls the shape of the pushed message, such as its fields and metadata. The endpoint version is based on the version of the Pub/Sub API. If not present during the `CreateSubscription` call, it defaults to the version of the API used to make such call. If not present during a `ModifyPushConfig` call, its value will not be changed. `GetSubscription` calls always return a valid version, even if the subscription was created without this attribute. The possible values for this attribute are `v1beta1`, which uses the push format defined in the v1beta1 Pub/Sub API, or `v1beta2`, which uses the push format defined in the v1 Pub/Sub API. | NO | - |
```

Comment:
```suggestion
| **pushConfig.attributes** | `object` | Endpoint configuration attributes. Every endpoint has a set of API supported attributes that you can use to control different aspects of the message delivery. The currently supported attribute is **x-goog-version**, which you can use to change the format of the pushed message. This attribute indicates the version of the data expected by the endpoint. This controls the shape of the pushed message, such as its fields and metadata. The endpoint version is based on the version of the Pub/Sub API. If not present during the **CreateSubscription** call, it defaults to the version of the API used to make such call. If not present during a **ModifyPushConfig** call, its value remains unchanged. **GetSubscription** calls always return a valid version, even if the subscription was created without this attribute. The possible values for this attribute are `v1beta1`, which uses the push format defined in the v1beta1 Pub/Sub API, or `v1beta2`, which uses the push format defined in the v1 Pub/Sub API. | NO | - |

```

I would use bold for the calls since these are attributes/parameters and we mention also their values.